tests/unit/test_simulated_stack.py

```diff
@@ -334,6 +334,29 @@ def testGetVariantSet(self):
         for badId in self.getBadIds():
             self.verifyGetMethodFails(path, badId)
 
+    def testGetVariant(self):
+        # get a variant from the search method
+        referenceName = '1'
+        start = 0
+        dataset = self.backend.getDatasets()[0]
+        variantSet = dataset.getVariantSets()[0]
+        request = protocol.SearchVariantsRequest()
+        request.variantSetId = variantSet.getId()
+        request.referenceName = referenceName
+        request.start = start
+        request.end = 2**16
+        path = '/variants/search'
+        responseData = self.sendSearchRequest(
+            path, request, protocol.SearchVariantsResponse)
+        variant = responseData.variants[0]
+
+        # get 'the same' variant using the get method
+        path = '/variants'
+        responseObject = self.sendGetObject(
+            path, variant.id, protocol.Variant)
+        self.assertEqual(responseObject.referenceName, referenceName)
+        self.assertEqual(responseObject.start, start)
```

Comment:
The ids don't come back the same, but I guess that means there's an issue with our random number generation... I'll look into it...